tools/site_and_regional/default_data.cfg

```diff
@@ -0,0 +1,28 @@
+[main]
+clmforcingindir = /glade/p/cesmdata/inputdata
+
+[datm_gswp3]
+dir = /glade/p/cgd/tss/CTSM_datm_forcing_data/atm_forcing.datm7.GSWP3.0.5d.v1.c170516
+domain = domain.lnd.360x720_gswp3.0v1.c170606.nc
+solardir = Solar
+precdir = Precip
+tpqwdir = TPHWL
+solartag = clmforc.GSWP3.c2011.0.5x0.5.Solr.
+prectag = clmforc.GSWP3.c2011.0.5x0.5.Prec.
+tpqwtag = clmforc.GSWP3.c2011.0.5x0.5.TPQWL.
+solarname = CLMGSWP3v1.Solar
+precname = CLMGSWP3v1.Precip
+tpqwname = CLMGSWP3v1.TPQW
+
+[surfdat]
+dir = lnd/clm2/surfdata_map/release-clm5.0.18
+surfdat_16pft = surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_c190214.nc
+surfdat_78pft = surfdata_0.9x1.25_hist_78pfts_CMIP6_simyr2000_c190214.nc
+
+[landuse]
+dir = lnd/clm2/surfdata_map/release-clm5.0.18
+landuse_16pft = landuse.timeseries_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr1850-2015_c190214.nc
+landuse_78pft = landuse.timeseries_0.9x1.25_hist_78pfts_CMIP6_simyr1850-2015_c190214.nc
+
+[domain]
+file = share/domains/domain.lnd.fv0.9x1.25_gx1v7.151020.nc
```

Comment:
I think this makes sense.  Will this work on other systems?  Do we need to consider this down the road?

Reply:
In a future PR my plan is actually to change these methods to something like `--from-compset` and/or `--from-resolution`. We would then create a dummy case under the hood to find the location of the files needed for subsetting that specific resolution and case. Otherwise, we would need to create methods to deal with running this on a machine other than Cheyenne.

Reply:
Yes, it should work on other systems. The important part of that is @jedwards4b suggestion below on setting the path  to DIN_LOC_ROOT.

Reply:
Yes, I'm going to resolve this conversation since as @adrifoster points out it'll be handled in #1597